versions/kruise/next/templates/daemonset.yaml

```diff
@@ -0,0 +1,149 @@
+{{-  if not (contains "KruiseDaemon=false" .Values.featureGates) }}
+{{- if .Values.installation.daemonSet.createNamespace }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.installation.daemonSet.namespace }}
+{{- end }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kruise-daemon
+{{- if .Values.serviceAccount.annotations }}
+  annotations:
+{{ toYaml .Values.serviceAccount.annotations | indent 4 }}
+{{- end }}
+  namespace: {{ .Values.installation.daemonSet.namespace }}
+{{ ( include "serviceAccountDaemon" . ) }}
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kruise-daemon
+  namespace: {{ .Values.installation.daemonSet.namespace }}
```

Comment:
the daemonset installation namespace is not the same as the configuration namespace, plz keep the namespace as  is (.Values.installation.namespace)

Reply:
I get what you're saying but it's still unclear to me the purpose of a separate namespace. What is it for?